Tests/LibWeb/Text/input/wpt-import/css/css-scroll-snap/support/common.css

```
@@ -0,0 +1,44 @@
+body {
+  margin: 0;
+}
+
+#scroller {
+  position: absolute;
+  width: 400px;
+  height: 400px;
+  overflow: scroll;
+  padding: 0;
+
+  scroll-snap-type: both mandatory;
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)